test/cookie_coder_test.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "test_helper"
+
+describe Identity::CookieCoder do
+  secret = "Pw/Xv58CsV8FVUQldtbHRZBoGqv8pJ1u55JdlhsRp9A="
+  another_secret = "r1auRylYrR9WM3dhRkfXRwQ5nxcNGR2lBkCLgXl94gU="
+
+  it "encryptes a cookie, then descrypts a cookie" do
+    data = {
+      user: {
+        id: "1234",
```

Comment:
It's not necessary. Just in case it's needed. This is a user UUID. We put it in identity's own cookie (id.heroku.com). I can remove it if it's not needed.